Cargo.toml

```diff
@@ -50,30 +50,30 @@ coretime-kusama-emulated-chain = { path = "integration-tests/emulated/chains/par
 coretime-kusama-runtime = { path = "system-parachains/coretime/coretime-kusama" }
 coretime-polkadot-emulated-chain = { path = "integration-tests/emulated/chains/parachains/coretime/coretime-polkadot" }
 coretime-polkadot-runtime = { path = "system-parachains/coretime/coretime-polkadot" }
-cumulus-pallet-aura-ext = { version = "0.17.0", default-features = false }
-cumulus-pallet-parachain-system = { version = "0.17.1", default-features = false }
-cumulus-pallet-session-benchmarking = { version = "19.0.0", default-features = false }
-cumulus-pallet-xcm = { version = "0.17.0", default-features = false }
-cumulus-pallet-xcmp-queue = { version = "0.17.0", default-features = false }
-cumulus-primitives-aura = { version = "0.15.0", default-features = false }
-cumulus-primitives-core = { version = "0.16.0", default-features = false }
-cumulus-primitives-utility = { version = "0.17.0", default-features = false }
-emulated-integration-tests-common = { version = "16.0.0" }
-encointer-balances-tx-payment = { version = "~14.1.0", default-features = false }
-encointer-balances-tx-payment-rpc-runtime-api = { version = "~14.1.0", default-features = false }
+cumulus-pallet-aura-ext = { version = "0.18.0", default-features = false }
+cumulus-pallet-parachain-system = { version = "0.18.0", default-features = false }
+cumulus-pallet-session-benchmarking = { version = "20.0.0", default-features = false }
+cumulus-pallet-xcm = { version = "0.18.0", default-features = false }
+cumulus-pallet-xcmp-queue = { version = "0.18.0", default-features = false }
+cumulus-primitives-aura = { version = "0.16.0", default-features = false }
+cumulus-primitives-core = { version = "0.17.0", default-features = false }
+cumulus-primitives-utility = { version = "0.18.0", default-features = false }
+emulated-integration-tests-common = { version = "19.0.0" }
```

Comment:
Did the PSVM tool work or how did you figure those out?

Reply:
It did work. I used `psvm -v stable2412-1`